cards/cards.css

```diff
@@ -0,0 +1,127 @@
+@import "../for-import/reset.css"; 
+@import "../for-import/colors.css";
+@import "../for-import/temp-buttons.css";
+@import "../for-import/typography.css";
+
```

Comment:
For whole file, indentation can be taken care of.
.class-name {
4spaces property: value;
}

.class-name2 {
    color: red;   
}

Reply:
done